hanadb_exporter/exporters/prometheus_exporter.py

```diff
@@ -45,10 +45,10 @@ def _manage_gauge(self, metric, formatted_query_result):
                 # TODO: exception labels not found
                 # TODO: exception value not found
                 try:
-                    labels.insert(metric.labels.index(column_name), column_value)
+                    labels.insert(metric.labels.index(column_name), column_value.lower())
```

Comment:
Have you tested this? I think that `index` method will filter depending if the string is upper or lowercase. 
We would need to convert all labels imported from metrics file to lowercase first (I would do that in prometheus_metrics.py file) and the as you do convert the received value too